native/app/components/DrawerItems.js

```diff
@@ -78,10 +71,10 @@ class DrawerItems extends React.Component {
           borderColor='#FFFFFF'
           borderWidth={2}
           resizeMode='cover'
-          source={{uri: this.state.profileAvatar}}
+          source={{uri: this.props.state.sync.avatar}}
```

Comment:
@sebastienbarbier i had issues here where if this is ever undefined (passed into `uri` then it all show a red error). I am not sure if we need to do a check earlier in the `jsx` or not

Reply:
This should not be undefined ... I'll investigate about that but had no issue for now.